react/features/welcome/actions.js

```diff
@@ -0,0 +1,17 @@
+import { SET_SIDEBAR_VISIBILITY } from './actionTypes';
+
+/**
+ * Redux action to hide or show the status bar.
+ *
+ * @param {boolean} visible - The new value of the visibility.
+ * @returns {{
+ *  type: SET_SIDEBAR_VISIBILITY,
+ *  sideBarVisible: boolean
+ * }}
+ */
+export function setSideBarVisibility(visible) {
+    return {
+        type: SET_SIDEBAR_VISIBILITY,
+        sideBarVisible: visible
```

Comment:
better call this "visible", the fact that we are talking abut the sidebar is implied by the action type

Reply:
Yeah, but this is the overall state of the WelcomeScreen, so a simple boolean 'visible' doesn't tell what is visible, when accessing the welcome screen state somewhere else.

Reply:
E.g. it'll be not clear what is supposed to be visible when you see this in a _mapStateToProps call:
state['features/welcome'].visible